### DISCUSSION TITLE
How NuGet packages are built?

### DISCUSSION THREAD

**Discussion by lkallas** [edenhill/librdkafka#3617] [Q&A]
I've forked `librdkafka` and done some code changes. How do I build my own NuGet package?
Can someone provide documentation for it and steps that need to be followed. Would really appreciate it.

**Comment by alexander-smyslov** [comment]
Use vcpkg 

vcpkg install --triplet x64-windows  librdkafka[core,ssl,zlib,snappy,zstd] 
vcpkg export --triplet x64-windows  librdkafka[core,ssl,zlib,snappy,zstd] --nuget --nuget-id=lib --nuget-version=1.0.0 --debug

**Reply by RPG-18** [reply]
Thanks for updating the version.
